applications/better_craigslist.md

```diff
@@ -0,0 +1,153 @@
+# Enhancing zkEmail with Plonky3: Advanced Features for Decentralized Systems
+## [Section 1] Project Information
+
+Project Name: Enhancing zkEmail with Plonky3: Advanced Features for Decentralized Systems
+Payment Details: 0x0ed18cFf1e16Db3f8b76D05c84182E4849ab03D4
+Total Amount Requested: 15,000 USDC
+
+## [Section 2] Project Overview :page_facing_up:
+
+Brief Description:
+This project aims to enhance zkEmail by developing new features and improvements using Plonky3, specifically tailored for decentralized systems with a focus on C2C marketplaces. We will expand zkEmail's capabilities to address data integrity, censorship resistance, and privacy, while pushing the boundaries of Plonky3's application in practical scenarios.
+
+Core Idea:
+The primary goal is to extend zkEmail's functionality by implementing the following use cases:
+
+- Enhanced data integrity verification for email attachments using Plonky3
+  - Experiment with following attachment formats: PDF, DOCX, PNG, JPEG
```

Comment:
I think attachment can be treated as binary data and it isn't depend on file extension. The most hard part is proving the attachment is a part of email.

Why do we need to generate proof for every attachment instead of create a proof for the whole email if we don't receive any benefit from it?.

Reply:
The idea is actually to have a circuit that decodes the attachment and then can perform some operation in it. For this we will start with the base case for PDF, and DOCX as we can then use zk-regex to prove that the attachment had some statement that we want to prove. It is something that is also pointed out on the repo as a Research Idea (Attachment Parsing Circuit) and this opens a lot of possibilities. Is also pointed out as having some performance bottlenecks so we want to try with plonky3 and do some research on how we can remove this performance issue

Reply:
The attachment is actually binary data on the email body, it is encoded into base64